packages/app/src/pages/endpoints/[endpointId]/_/content/types/table/index.tsx

```diff
@@ -174,3 +205,143 @@ const Operations: React.FC<OperationsProps> = ({
     </>
   );
 };
+
+const RowObject: React.FC<
+  BaseProps & { rowObject: Data; columns: TableColumn[]; close: () => void }
+> = ({ on, rowObject, columns, close }) => {
+  return (
+    <div className="flex flex-col h-full w-full">
+      <div className="px-10 pt-10">
+        <div className="text-right">
+          <Button
+            variant="text"
+            on={COLOR_SYSTEM.SURFACE}
+            Icon={XIcon}
+            onClick={close}
+          />
+        </div>
+      </div>
+      <div className="px-10 pb-10 grow overflow-scroll">
+        <div>
+          {Object.keys(rowObject).map((objectKey, index) => (
+            <div className="mt-10">
```

Comment:
If the parent have space-y class, mt-10 and the div can be deleted.

Reply:
I will adopt your suggesion.